.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python:
+  - 3.6
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/.ccache  # https://github.com/travis-ci/travis-ci/issues/5853
+
+install:
+  # Install this package and the packages listed in requirements.txt.
+  - pip install .
+  # Install extra requirements for running tests and building docs.
+  - pip install -r requirements-dev.txt
```

Comment:
```suggestion
```

Reply:
I love this new GH feature. I think you should probably also remove the line above with the comment.

Reply:
I haven't seen that feature before, but it does look good. I will have to take a look.

thanks @mrakitin for indicating the issue, I have removed this in the latest commit.